wire-ios-data-model/Source/Model/Message/ConversationMessage.swift

```diff
@@ -340,14 +342,37 @@ extension ZMMessage {
         return .delivered
     }
 
-    @objc public var usersReaction: [String: [UserType]] {
-        var result = [String: [ZMUser]]()
+    @objc public var reactionData: Set<ReactionData> {
+        var result = Set<ReactionData>()
         for reaction in reactions where reaction.users.count > 0 {
-            result[reaction.unicodeValue!] = [ZMUser](reaction.users)
+            result.insert(
+                ReactionData(
+                    reactionString: reaction.unicodeValue!,
+                    users: reaction.users.map { $0 },
```

Comment:
Why not `users: reaction.users`?

Reply:
It's a change from `Set` to `Array`. I could write it as `Array(reaction.users)` if you think thats better.

Reply:
Yes I think it's more expressive to say `Array(reaction.users)`.